tianshou/env/gym_wrappers.py

```diff
@@ -0,0 +1,31 @@
+import gym
+import numpy as np
+
+
+class DiscreteToContinuous(gym.ActionWrapper):
+    """Gym environment wrapper to take discrete action in a continous environment.
+
+    Args:
+        env (gym.Environment): gym envirionment with continous action space.
+        action_per_branch (int): number of discrete actions in each dimension of the action space.
+    
+    """
+
+    def __init__(self, env: gym.Env, action_per_branch: int) -> None:
+        super().__init__(env)
+        self.action_per_branch = action_per_branch
+        low = env.action_space.low
+        high = env.action_space.high
+        num_branches = env.action_space.shape[0]
+        self.action_space = gym.spaces.MultiDiscrete(
```

Comment:
How would you handle the MultiDiscrete action space with current codebase?

Reply:
This is a class for using discrete actions in continuous spaces. It works with the BDQ algorithm.
For example if you have a continuous action space as two intervals ([0, 3], [6, 9]), you can specify the number of bins you want to divide the continuous intervals into, let's say 4 (this is the actions_per_branch). Then we'll have a new multi-discrete space of the form ([4, 4]). And this wrapper let's you pass discrete actions for example [0, 2] from the multi-discrete space,  which will correspond to [0, 8] in the continuous space.